cmd/vicadmin/vicadm.go

```diff
@@ -405,29 +407,53 @@ func (r datastoreReader) open() (entry, error) {
 	return httpEntry(r.path, res)
 }
 
+// removes user credentials from "in"
```

Comment:
Ah, I meant to say that we should replace `"in"` with `session`. 😃

Just a tiny detail, we can leave it as-is.